app/templates/macros/tables/stav_registraci_kategorie.html

```diff
@@ -0,0 +1,48 @@
+{% macro stav_registraci_kategorie(reservations_category) -%}
+
+<div class="table-responsive">
+    <table class="table table-bordered table-striped table-sm mb-0" style="table-layout: fixed">
+        <thead class="thead-dark">
+        <tr>
+            <th scope="col" class="w-20 text-center" rowspan="1">Kategorie</th>
```

Comment:
we use `Prioritní skupina` in the simlar table, at least `Skupina` would be better?